sysutils/ondir/files/patch-Makefile.diff

```diff
@@ -0,0 +1,36 @@
+--- Makefile.orig	2019-06-27 19:33:59.000000000 +0200
++++ Makefile	2019-06-27 19:37:49.000000000 +0200
+@@ -1,6 +1,6 @@
+ # Change these as you see fit
+-PREFIX=/usr
+-CONF=/etc/ondirrc
++PREFIX=/opt/local
```

Comment:
You may not want to hardcode the prefix directly like so; some folks install MacPorts to a prefix that is different than `/opt/local`.  Perhaps you can have the patch use a placeholder like `PREFIX=@@PREFIX@@`, and then in your Portfile, in the post-patch phase, use `reinplace` to replace `@@PREFIX@@` with `${prefix}`.

This is actually mentioned in the MacPorts wiki: https://trac.macports.org/wiki/PortfileRecipes#hardcode_opt_local

Reply:
ciao @herbygillot, thank you very much for your suggestion. should be fixed now.
(actually, i have missed that wiki page, i'll go through the whole of it)
cheers,
--
ferdy

Reply:
Thanks for addressing that @ferdy.  I haven't tried building it, but it looks good.  Don't forget to squash your changes into one commit.

Reply:
should be done as well now. thanks again for your help and guidance! cheers,
\--
ferdy